config/settings/sentry.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+import sentry_sdk
+from config.util import strtobool
+from config.versioning import get_git_sha
+from django.conf import settings
+from sentry_sdk.integrations.django import DjangoIntegration
+
+if SENTRY_DSN := os.getenv("SENTRY_DSN", ""):
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        environment=os.getenv("ENVIRONMENT_NAME", "production"),
+        release=get_git_sha(),
+        integrations=[DjangoIntegration()],
+        sample_rate=settings.SENTRY_SAMPLE_RATE,
+        traces_sample_rate=settings.SENTRY_SAMPLE_RATE,
+        profiles_sample_rate=settings.SENTRY_SAMPLE_RATE,
+    )
```

Comment:
It feels wrong to have executable code mixed in with configuration. This also means we have to have the weird:

```python
try:
    SENTRY_SAMPLE_RATE = float(os.getenv("SENTRY_SAMPLE_RATE", "0.1"))
    from .sentry import *
except ImportError:
    pass
```

Instead, would this code be better placed in `app/__init__.py`? That code would be executed on every application startup, right?